doc/runtime.rst

```diff
@@ -130,3 +139,11 @@ The user can set these environment variables to adjust command execution behavio
 ``SR_CMD_TIMEOUT`` should be specified in seconds.  Note that ``SR_CMD_INTERVAL``
 and ``SR_CMD_TIMEOUT`` are read during client initialization and not
```

Comment:
Isn't this behavior true of all of the environment variables for the client?

Reply:
We don't currently have an overview of environment variables, but maybe we should add a table to the top of this text that has overview info like this. Not something that can fit into this cycle though